notebooks/official/automl/automl_forecasting_on_vertex_pipelines.ipynb

```diff
@@ -0,0 +1,1867 @@
+{
```

Comment:
<p>Here, we explain the parameters for TiDE and L2L. What about TFT / Seq2Seq+? If the parameters are the same for all four model types, we should say "The utility functions for all model types have the same arguments". </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/2184/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='46'/>

Reply:
<p>Fixed.</p>

Reply:
<p>I'm not sure if you made the change elsewhere, but the text "The utility functions of both <code>L2L</code> and <code>TiDE</code> model type have the same arguments" is still the same. </p>

Reply:
<p>Just pushed the change, it should show up now.</p>